api/controllers/v1/slack.ts

```diff
@@ -66,7 +81,7 @@ async function deployRepo(deployable: Array<any>, logger: ILogger, jobRepository
   try {
     await jobRepository.insertBulkJobs(deployable, jobQueueUrl);
   } catch (err) {
-    logger.error('deployRepo', err);
+    console.error('Deploy repo error');
```

Comment:
Could we still log the error? It can sometimes be such a blessing.

Reply:
Happy to, but it won't make a difference in this case! The only place we can view the logs for the Slack Deploy Repo lambda is in its corresponding CloudWatch group, in which the output for console.error looks like the following screenshot:
<img width="1088" alt="Screenshot 2024-05-30 at 2 52 48 PM" src="https://github.com/mongodb/docs-worker-pool/assets/41971124/19147203-25d2-477e-bcbc-4e61ea2bc14f">

Reply:
Fascinating! I wonder if that's always the case... I lean towards keeping the error logged just in case it can sometimes be helpful. But good catch!